silkworm/core/execution/evm.cpp

```diff
@@ -349,19 +330,15 @@ evmc_result EVM::execute_with_baseline_interpreter(evmc_revision rev, const evmc
         }
     }
     if (!analysis) {
-        analysis = std::make_shared<evmone::baseline::CodeAnalysis>(evmone::baseline::analyze(rev, code));
+        const bool eof_enabled{rev >= EVMC_PRAGUE};
```

Comment:
It looks like EOF will be postponed until PectrB (=Osaka?)

Reply:
This reflects the status of evmone, but actually we can put `false` here for now.

Reply:
Set to `false` with a comment.